lib/schemas/store-order-recipient.js

```diff
@@ -1,16 +1,43 @@
+const { every } = require('lodash');
 const yup = require('yup');
+
 const config = require('../config');
 
+/**
+ * Helper that returns boolean given a value
+ * Returns true if value is a positive number or UUID
+ *
+ * @param {*} value Value being tested
+ *
+ * @returns {*} Boolean
```

Comment:
Nit:
```suggestion
 * Predicate validation helper for Reward Ops item IDs.
 *
 * @param {*} value Value being tested
 *
 * @returns {boolean} True if value is a positive number or UUID, false otherwise
 *
 * @protected
```